packages/docusaurus-theme-translations/locales/base/theme-common.json

```diff
@@ -83,7 +83,7 @@
   "theme.docs.breadcrumbs.home___DESCRIPTION": "The ARIA label for the home page in the breadcrumbs",
   "theme.docs.breadcrumbs.navAriaLabel": "Breadcrumbs",
   "theme.docs.breadcrumbs.navAriaLabel___DESCRIPTION": "The ARIA label for the breadcrumbs",
-  "theme.docs.paginator.navAriaLabel": "Docs pages navigation",
+  "theme.docs.paginator.navAriaLabel": "Docs pages",
```

Comment:
Could you try to update some of the localization messages, and for the rest that you are not sure, revert them to the English one?

Reply:
Sure, I've updated the translations